tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-initial-dialog.html

```diff
@@ -28,24 +29,36 @@
     <paper-dialog id="dialog"
                   modal
                   opened="{{_open}}"
-                  width="320"
                   with-backdrop="[[_useNativeBackdrop]]">
-      <h2 id='dialog-title'>Waiting for first Session.run() to connect...</h2>
+      <h2 id='dialog-title'>Debugger is waiting for Session.run() connections...</h2>
       <div class='code-example'>
-        <!-- TODO(cais): Rename id. -->
-        <pre id="session-run-code-example">
-# To connect to the debugger from your tf.Session:
-from tensorflow.python import debug as tf_debug
-sess = tf.Session()
-sess = tf_debug.TensorBoardDebugWrapperSession(sess, "[[_host]]:[[_port]]")
-sess.run(my_fetches)
-        </pre>
-        <pre id="hook-code-example">
-# To connect to the debugger using hooks, e.g., from tf.Estimator:
-from tensorflow.python import debug as tf_debug
-hook = tf_debug.TensorBoardDebugHook("[[_host]]:[[_port]]")
-my_estimator.fit(x=x_data, y=y_data, steps=1000, monitors=[hook])
-        </pre>
+        <div class="code-example-section">
+          <a href="https://www.tensorflow.org/api_docs/python/tf/Session" target="_blank">tf.Session</a>s:
```

Comment:
Hmm, this used to say "To connect to the debugger from your tf.Session:" but now we just get "tf.Session:" - I think a little more instruction might be clearer?  E.g. perhaps just something like:

To connect with a tf.Session():
To connect with an Estimator and MonitoredSession:
To connect with a Keras model: